crates/driver/solvers.example.toml

```diff
@@ -0,0 +1,12 @@
+[[solvers]]
+name = "mysolver" # Arbitrary name given to this solver, must be unique
+url = "http://localhost:1234/solve" # Endpoint of the solver
+absolute-slippage = 12 # Denominated in wei, optional
+relative-slippage = 0.1 # Percentage in the [0, 1] range
```

Comment:
Should passing something outside the range of `[0, 1]` throw an error? This case is currently not checked.